examples/tutorials/Part 11b - Secure Classification with Syft Keras and TFE: Secure Model Serving.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<p>num_steps is a poorly named variable. I get what it means but the first time user probably won't. We need to specify that the server is expecting to serve only a limited number of predictions.</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>